### PR TITLE
Compatibility.py: Add comment

### LINKED ISSUE #4156: Compatibility.py: Add comment explaining JSONDecodeError is missing in Python 3.4
difficulty/newcomer

Opened via [gitter](https://gitter.im/coala/coala/?at=59098da68fcce56b205cd7e0) by @jayvdb

### PR DESCRIPTION
**Pull request by shreyateeza**
Add comment explaining JSONDecodeError is available since Python 3.5.x.

Closes #4156

**For short term contributors:** we understand that getting your commits well
defined like we require is a hard task and takes some learning. If you
look to help without wanting to contribute long term there's no need
for you to learn this. Just drop us a message and we'll take care of brushing
up your stuff for merge!

### Checklist

- [x] I read the [commit guidelines](http://coala.io/commit) and I've followed
      them.
- [x] I ran coala over my code locally. (*All commits have to pass
      individually.* It is not sufficient to have "fixup commits" on your PR,
      our bot will still report the issues for the previous commit.) You will
      likely receive a lot of bot comments and build failures if coala does not
      pass on every single commit!

After you submit your pull request, **DO NOT click the 'Update Branch' button.**
When asked for a rebase, consult [coala.io/rebase](https://coala.io/rebase)
instead.

Please consider helping us by reviewing other peoples pull requests as well:

- pick up any PR at <https://coala.io/review>
- review it (check <https://coala.io/reviewing> for more info)
- if you are sure that it needs work, use `cobot mark wip <URL>` to get it out
  of the review queue.

The more you review, the more your score will grow at coala.io and we will
review your PRs faster!

### CODE REVIEW

**Comment by shreyateeza** [comment]
@Makman2 is it alright now? Also I apologise for creating a new pull request. I had to make one due to some unavoidable circumstances.

File: coalib/misc/Compatibility.py [hunk 1]
```diff
@@ -1,5 +1,6 @@
 import json
 try:
+    # JSONDecodeError class is available starting since Python 3.5.x.
```
Comment (Makman2) [inline coalib/misc/Compatibility.py]: either
"... is available since"
or
"... is available starting from"
:D

**Comment by Makman2** [comment]
unack c8dc8b6

**Comment by Makman2** [comment]
ack 565dedf

**Comment by Makman2** [comment]
@gitmate-bot rebase

**Comment by jayvdb** [comment]
ack https://github.com/coala/coala/pull/4980/commits/15decc024441248d3d1fcb40b95e540254afc296

**Pull request merged**